### ISSUE TITLE
Add Wiimmfi to dns so you can play online in wii mode

### ISSUE THREAD

**Issue by owen-the-kid** [PretendoNetwork/SSSL-DNS#7]
Basicly what the title said.
Riiconnect24 also had this feature.

**Comment by JPeisach** [comment]
I don't think this is possible as these are fundamentally different systems and vWii is separate from the Wii U modules.

**Comment by owen-the-kid** [comment]
vwii uses the same config as the wii and my wiiu can use wiimmfi

**Comment by JPeisach** [comment]
🤔 this would work, depends on if vWii certificates are handled similarly to Wii U certificates

**Comment by owen-the-kid** [comment]
i think it could work

**Comment by InternalLoss** [comment]
This is possible and planned, as Nintendo Wi-Fi Connection uses a different set of domains. We’re working on this internally - many thanks for your patience! :heart:

**Comment by JPeisach** [comment]
Is it a similar tactic that exists in regular Wii firmware?
